From d62a3e6540d0f8b2ff5444ba003e2074a714c452 Mon Sep 17 00:00:00 2001
From: Andre Przywara <andre.przywara@arm.com>
Date: Wed, 16 Jun 2021 18:37:38 +0100
Subject: [PATCH 23/23] arm64: dts: allwinner: h616: Remove 32K clock

The RTC in the new Allwinner H616 SoC has some extra "features",
also it does not support a 32768 Hz oscillator input anymore.

Add the DT node describing it and change the preliminary fixed clock
to be the actual RTC provided clock.

Signed-off-by: Andre Przywara <andre.przywara@arm.com>
---
 .../arm64/boot/dts/allwinner/sun50i-h616.dtsi | 26 ++++++++++---------
 1 file changed, 14 insertions(+), 12 deletions(-)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-h616.dtsi b/arch/arm64/boot/dts/allwinner/sun50i-h616.dtsi
index 7681f2a68dee..c55933f32d99 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-h616.dtsi
+++ b/arch/arm64/boot/dts/allwinner/sun50i-h616.dtsi
@@ -70,13 +70,6 @@ osc24M: osc24M_clk {
 		clock-output-names = "osc24M";
 	};
 
-	osc32k_int: osc32k-int {
-		#clock-cells = <0>;
-		compatible = "fixed-clock";
-		clock-frequency = <32768>;
-		clock-output-names = "osc32k-int";
-	};
-
 	pmu {
 		compatible = "arm,cortex-a53-pmu";
 		interrupts = <GIC_SPI 140 IRQ_TYPE_LEVEL_HIGH>,
-- 
2.25.1

